cmd/marketplace/main.go

```diff
@@ -26,11 +27,29 @@ func main() {
 		return
 	}
 
+	ctx := context.Background()
+
+	cfg, err := marketplace.NewConfig(*configPath)
+	if err != nil {
+		log.GetLogger(ctx).Error("failed to load config", zap.Error(err))
+		os.Exit(1)
+	}
+
+	key, err := accounts.LoadKeys(cfg.Eth.Keystore, cfg.Eth.Passphrase)
```

Comment:
Consider creating generic EthConfig and put processing inside

Reply:
done